pkgs/development/python-modules/backports-zoneinfo/default.nix

```diff
@@ -69,6 +69,9 @@ buildPythonPackage rec {
     "test_folds_from_utc"
     # backports.zoneinfo._common.ZoneInfoNotFoundError: 'No time zone found with key Eurasia/Badzone'
     "test_bad_keys"
+    # (on python38) AssertionError: datetime.timedelta(days=-1, seconds=84879) != datetime.timedelta(days=-1, seconds=84900) : 1880-07-31 00:00:00-00:25:21
```

Comment:
Do these only fail on python 3.8?

Reply:
Mhm

Reply:
We no longer support python 3.8

Reply:
`hexa` on matrix told me the same. Remove or keep? Apparently the python date packages consistently have tests like this.

Reply:
My point was just to only disable these packages conditionally for python<=3.8

Reply:
Ended up dropping, ran my exact-I-hope thing from before but it doesn't reproduce. Maybe I rebased onto master.